homeassistant/components/geonetnz_quakes/geo_location.py

```diff
@@ -14,6 +14,7 @@
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.util.unit_system import IMPERIAL_SYSTEM
 
+from ...helpers.entity_registry import async_get_registry
```

Comment:
Please make this an absolute import.

Reply:
OK. PyCharm did that, but it also had a shortcut to turn this back into an absolute import for me. :)